### PR TITLE
DOCS-9146 fixed table issues in policies

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 1]
```diff
@@ -3,22 +3,21 @@ ifndef::env-site,env-github[]
...
+.3+>.^s| Returned Status Codes |
 
 [%autowidth.spread,cols="a,a"]
 !===
-.2+>.^! 400 ! A token is invalid, or their detokenization failed.
-! The provided expression is invalid or returns no result to detokenize. +
-*NOTE*: The status code to be returned can be configured.
-! 502 ! Any problem regarding the connection with the tokenization service. It may be unavailable, or it might not have been configured.
-!===
+.3+>.^! 400 ! A token is invalid, or their detokenization failed. ! The provided expression is invalid or returns no result to detokenize.
```
Comment (dhirsch1) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: What does "their" refer to?

File: modules/ROOT/pages/policy-mule4-detokenization.adoc [hunk 2]
```diff
@@ -3,22 +3,21 @@ ifndef::env-site,env-github[]
...
-! 502 ! Any problem regarding the connection with the tokenization service. It may be unavailable, or it might not have been configured.
-!===
+.3+>.^! 400 ! A token is invalid, or their detokenization failed. ! The provided expression is invalid or returns no result to detokenize. 
 
+*NOTE*: You can configure the status code to be returned.
+!===
+| 502 - Connection to the tokenization service is unavailable or not configured.
 |===
 
 Detokenization is the process of returning the previously masked sensitive data back into its original value to reduce the risk of compromising sensitive information. You can tokenization your data to replace sensitive information with tokens to reduce the risk of compromising sensitive information.
```
Comment (dhirsch1) [inline modules/ROOT/pages/policy-mule4-detokenization.adoc]: I know you didn't change this in this PR, but "You can tokenization" should probably be "You can tokenize..."

File: modules/ROOT/pages/policy-mule4-tokenization.adoc [hunk 3]
```diff
@@ -3,26 +3,26 @@ ifndef::env-site,env-github[]
...
+.3+>.^!400 ! A token is invalid, or the tokenization failed. ! The provided expression is invalid or returns no result to tokenize. 
+! The status code to be returned can be configured.
 !===
-
+| 502 - Connection to the tokenization service is unavailable or not configured.
 |===
  
 Tokenization is the process of masking a value or piece of information that can be considered sensitive data into a token that can be mapped back to its original value using detokenization concepts. If your API contains sensitive data, the tokenization policy is a highly effective way to protect it. 
 
+The data is displayed in the same format as the original value, but its actual value is not revealed at first sight. This policy is designed to work only for Anypoint Runtime Fabric deployments.
```
Comment (dhirsch1) [inline modules/ROOT/pages/policy-mule4-tokenization.adoc]: Maybe change "at first sight" > "initially"

**Review by mtqp — DISMISSED** [review]

File: modules/ROOT/pages/policy-mule4-detokenization.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule4-detokenization.adoc]: Good catch!

Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule4-detokenization.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule4-detokenization.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule4-detokenization.adoc]: Again a great catch! Done! Thanks!

File: modules/ROOT/pages/policy-mule4-tokenization.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule4-tokenization.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule4-tokenization.adoc [hunk 4]
```diff
@@ -3,26 +3,26 @@ ifndef::env-site,env-github[]
...
-.4+>.^s| Returned Status Codes |
+.3+>.^s| Returned Status Codes |
 
 [%autowidth.spread,cols="a,a"]
 !===
-.2+>.^! 400 ! A token is invalid, or the tokenization failed.
-! The provided expression is invalid or returns no result to tokenize. +
-The status code to be returned can be configured.
-! 502 ! Any problem regarding the connection with the tokenization service, such as the service is unavailable, or the service is not configured.
+.3+>.^!400 ! A token is invalid, or the tokenization failed. ! The provided expression is invalid or returns no result to tokenize.
```
Comment (dhirsch1) [inline modules/ROOT/pages/policy-mule4-tokenization.adoc:16]: Is the exclamation supposed to be there?

Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule4-tokenization.adoc:16]: Yes these are a part of the formatting.

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**